core/mutator/mutator.go

```diff
@@ -67,7 +67,8 @@ type MutationStorage interface {
 	// ReadAll reads all mutations starting from the given sequence number.
 	// Note that startSequence is not included in the result. ReadAll also
 	// returns the maximum sequence number read.
-	ReadAll(ctx context.Context, mapID int64, startSequence uint64) (uint64, []*pb.EntryUpdate, error)
+	// ReadAll will not return more than count entries.
```

Comment:
It seems like it might need a new name given that for suitable `count` values it doesn't read all mutations any more.
Maybe adding `count` is a hint that we now have a paginated read API?  So it could be called `ReadPage`?

Reply:
Or `ReadBatch` since you talk about batch sizes in the sequencer.

Reply:
`ReadRage` is becoming `ReadPage`, and `ReadAll` is becoming `ReadBatch`.  Is that too confusing?

Reply:
It's a bit confusing.  It feels like they're both doing the same thing really, and the main difference is whether it's the client doing a bit of math on start/end/count or the API implementation.  Does the client know currentSequence?

Another possibility might be for `count` not to be supplied through the API at all, but for the implementation to make it's own decision about whether to truncate output or not, and to indicate with a flag if truncation has occurred.  wdyt?